src/common/components/NavigationDrawer.vue

```diff
@@ -192,18 +213,24 @@ export default {
   },
 
   data: () => ({
-    bookIcon
+    bookIcon,
+    alertIcon,
+
+    version: ""
   }),
 
+  created() {
+    this.version = `v${process.env.VUE_APP_VERSION}`
```

Comment:
Don't you think it's better to get the version from `package.json` to minimalize maintaining version in 2 place?